.travis.yml

```diff
@@ -16,7 +16,7 @@ jdk: oraclejdk8
 rvm:
   - 2.4.2
   - 2.1.10
-  - jruby-9.1.14.0
+  - jruby-9.2.0.0
```

Comment:
Something that I've been doing is putting something like `jruby-9` so that it just gets whatever the newest version of jRuby 9.x.x.x is